Mathlib/NumberTheory/NumberField/Completion.lean

```diff
@@ -120,48 +120,48 @@ theorem subfield_ne_real_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
   obtain ⟨r, hr⟩ := hv ▸ extensionEmbedding_coe v x ▸ RingHom.mem_fieldRange_self _ _
   simp only [ComplexEmbedding.conjugate_coe_eq, ← hr, Complex.ofRealHom_eq_coe, Complex.conj_ofReal]
 
-/-- If `v` is a complex infinite place, then the embedding `v.completion →+* ℂ` is surjective. -/
+/-- If `v` is a complex infinite place, then the embedding `v.Completion →+* ℂ` is surjective. -/
 theorem surjective_extensionEmbedding_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
     Function.Surjective (extensionEmbedding v) := by
   rw [← RingHom.fieldRange_eq_top_iff]
   exact (Complex.subfield_eq_of_closed <| isClosed_image_extensionEmbedding v).resolve_left <|
     subfield_ne_real_of_isComplex hv
 
-/-- If `v` is a complex infinite place, then the embedding `v.completion →+* ℂ` is bijective. -/
+/-- If `v` is a complex infinite place, then the embedding `v.Completion →+* ℂ` is bijective. -/
 theorem bijective_extensionEmbedding_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
     Function.Bijective (extensionEmbedding v) :=
   ⟨(extensionEmbedding v).injective, surjective_extensionEmbedding_of_isComplex hv⟩
 
-/-- The ring isomorphism `v.completion ≃+* ℂ`, when `v` is complex, given by the bijection
-`v.completion →+* ℂ`. -/
+/-- The ring isomorphism `v.Completion ≃+* ℂ`, when `v` is complex, given by the bijection
+`v.Completion →+* ℂ`. -/
 def ringEquiv_complex_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
```

Comment:
This is also misnamed, but that's a problem for another PR, not this one.

Reply:
The logic here is that an infinite place `v` of a number field is either real (`NumberField.InfinitePlace.IsReal v : Prop`) or complex (`...IsComplex v`). In the real case,  `v.Completion` is (canonically) isomorphic to the reals, and in the complex case, `v.Completion` is (non-canonically) isomorphic to the complexes (the issue is that you can't distinguish the isomorphism and its complex conjugate). So "if the place is complex, we're isomorphic to the complexes". Do you still think it's misnamed @eric-wieser ?

Reply:
#19659